From d647872aee4871e286ddedf4931792086f5b4565 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@lantiq.com>
Date: Mon, 26 Oct 2015 14:32:39 +0100
Subject: [PATCH] libcoap: remove fix build error on big endian systems

In the big endian case an additional typedef is added in a wrong
position in TinyDTLS. This breaks compiling this code on big endian
systems.

Change-Id: Iad854aba112ddb23bf490b064ec6fbf5d01ce6b6
Signed-off-by: Hauke Mehrtens <hauke.mehrtens@lantiq.com>
---
 resource/csdk/connectivity/lib/libcoap-4.1.1/pdu.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/resource/csdk/connectivity/lib/libcoap-4.1.1/pdu.h
+++ b/resource/csdk/connectivity/lib/libcoap-4.1.1/pdu.h
@@ -194,7 +194,7 @@ typedef enum
 #ifdef WORDS_BIGENDIAN
 typedef union
 {
-    typedef struct
+    struct
     {
         unsigned int version:2; /* protocol version */
         unsigned int type:2; /* type flag */
